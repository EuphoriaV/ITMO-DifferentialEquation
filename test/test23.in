4.76
(((x - x) + (10.7 - 8.98)) - ((x * x) * (x * (x * 10.1))))
2.6
5.44