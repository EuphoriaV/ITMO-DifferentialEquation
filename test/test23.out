-5.544E-4